.buildkite/build_pipeline.yml

```diff
@@ -34,7 +38,7 @@ steps:
       provider: "gcp"
       image: family/docs-ubuntu-2204
       machineType: ${BUILD_MACHINE_TYPE}
-    concurrency_group: build-docs
+    concurrency_group: build-docs-${BUILDKITE_BRANCH}
```

Comment:
smart

Reply:
I realized the branch was not running because there was an on-going build on master :S